*** paths.go	2011-09-11 09:29:34.303683012 +0200
--- paths.win	2011-09-11 09:30:27.037199764 +0200
***************
*** 20,26 ****
  import "os"
  
  func GetConfFilenames() (conffile_etc string, conffile_home string) {
!         conffile_etc = "/etc/browserbridge/browserbridge.conf"
!         conffile_home = os.Getenv("HOME") + "/.browserbridge.conf"
  	return
  }
--- 20,26 ----
  import "os"
  
  func GetConfFilenames() (conffile_etc string, conffile_home string) {
!         conffile_etc = "C:\browserbridge.conf"
!         conffile_home = os.Getenv("APPDATA") + "\browserbridge.conf"
  	return
  }
